conf/application.conf

```diff
@@ -27,7 +27,7 @@ play.http.requestHandler = "uk.gov.hmrc.play.bootstrap.http.RequestHandler"
 # An audit connector must be provided.
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuditModule"
```

Comment:
there's probably some warnings for configuration that can be addressed.
Also, do we still need `HelloWorldFilters`?